### DISCUSSION TITLE
`extraPlugins.*.configLua`?

### DISCUSSION THREAD

**Discussion by mightyiam** [nix-community/nixvim#2457] [Q&A] [answered]
I'm trying to understand, please, why there is an [`extraPlugins.*.config`](https://nix-community.github.io/nixvim/NeovimOptions/extraPlugins/index.html#extrapluginsconfig) but not an equivalent `extraPlugins.*.configLua`.

And then I thought perhaps there's a helper that takes a lua string and returns a vimscript string that executes that lua string. But I don't see one...

Help?

**Comment by traxys** [comment, marked as answer]
This option is not managed by nixvim, but it's passed directly to nixpkgs: https://github.com/NixOS/nixpkgs/blob/25b642834b6921f4baf4fab4c7f63942e6736283/pkgs/applications/editors/neovim/wrapper.nix#L51-L55

To add such a feature it's more appropriate to open a nixpkgs issue

**Reply by mightyiam** [reply]
Thank you. I don't currently see a good enough reason to bother with that, given `extraConfigLua` seems to work just fine.
